app/scripts/controllers/courtSearchInfo.ctrl.js

```diff
@@ -11,7 +11,11 @@ angular.module('yourStlCourts').controller('CourtSearchInfoCtrl', function ($sta
     zoom: 14
   };
 
-  ctrl.courtMarkers = {};
+  leafletData.getMap("mapid").then(function(map){
```

Comment:
This doesn't seem appropriate in a controller. Can this just be set in the HTML or somewhere else?